.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Markdown Lint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: DavidAnson/markdownlint-cli2-action@v4
+      with:
```

Comment:
You have to use the npm package `markdownlint-cli2` not the action 😓

Reply:
> 

Sorry idk how to use the npm packages in GitHub actions, if there is any docs regarding this I will to change it 🥲

Reply:
You can check this one for reference (feel free to ask for any help on [our discord](https://krish.ninja/discord)): https://github.com/LearnWithCommunity/support/blob/main/.github/workflows/format.yml